script.js

```diff
@@ -1,4 +1,5 @@
 var main = function (input) {
   var myOutputValue = "hello papaya";
+  ;"hello";
```

Comment:
Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/rocketacademy/basics-ci-test/pullRequest?prid=7602387)
```suggestion
  "hello";
```